apps/communications/contacts/test/marionette/lib/contacts.js

```diff
@@ -386,10 +388,36 @@ Contacts.prototype = {
     }
   },
 
+  checkContactCreated: function() {
+    var ci = this.client.findElement(Contacts.Selectors.contactsIframe);
+    this.client.switchToFrame(ci);
+
+    var contactInContactList = this.client.findElement
+      (Contacts.Selectors.listContactFirst);
+    var contactName = contactInContactList.text();
+    return contactName;
+  },
+
   tapSave: function() {
     this.client.helper.waitForElement(Contacts.Selectors.formSave).click();
   },
 
+  tapSavefromDialer: function() {
```

Comment:
Nit: This function does more than just taping on the contact. I'd change its signature to:
```js
createNewContactAndReturnToDialer: function(name) {
```

What do you think?